Init (#1)
mate-academy/jv-web-practice: README.md

# Web practice. Implement JSP pages and their corresponding controllers

- Configure web.xml

> - Add new dependencies `servlet-api` and `maven-war-plugin` in `pom.xml` (remember about `<packaging>war</packaging>`)

Reviewer: Якщо це було частиною попередньої ДЗ то немає сенсу робити це ще раз

- Create a new controller: Implement creating a driver
- Create a new controller: Display all drivers
- Create a new controller: Implement creating a manufacturer
- Create a new controller: Implement creating a car (with no drivers by default)
- Create a new controller: Implement adding driver to car.

> HINT: in last controller, create just two fields on JSP where you will pass corresponding id values.

Reviewer: ```suggestion
HINT: while implementing the last requirement, create just two fields on JSP where you will pass corresponding id values.
```
